src/js/utils/video-fit.js

```diff
@@ -0,0 +1,36 @@
+import { Browser, OS } from 'environment/environment'; 
+import { style } from 'utils/css';
+
```

Comment:
If all of this code is coming out of html5, and into video-actions-mixin, put it there. There is no purpose to making a new "utils/video-fit.js" module. This is not a util. This code should just be part of resize in the mixin.